packages/core/src/core.ts

```diff
@@ -5,6 +5,8 @@ import {toCommandValue} from './utils'
 import * as os from 'os'
 import * as path from 'path'
 
+import {getIDTokenUrl, parseJson, postCall} from './oidc-utils'
```

Comment:
can we follow the same pattern as `artifacts` which is only importing the `OidcClient` out of `./oidc-utils`?
https://github.com/actions/toolkit/blob/main/packages/artifact/src/artifact-client.ts

and `OIDCHttpClient` only export a single method.
ex:

```js
export interface OidcClient {
  getIdToken(
    audience?: string,
  ): Promise<string>
```